From a7aac89f366c1ef1ca928751b08e11dc90cd1110 Mon Sep 17 00:00:00 2001
From: Vincent Palatin <vpalatin@chromium.org>
Date: Tue, 25 May 2021 16:08:10 +0200
Subject: [PATCH] gdbus: use /run/dbus rather than /var/run/dbus

When opening the D-Bus system socket, try '/run/dbus/system_bus_socket'
rather than '/var/run/dbus/system_bus_socket' as standardized by FHS
version 3.0.

Often both exist, but in some jails, '/var/run' is not bind-mounted
failing the D-Bus connection.
---
 gio/gdbusaddress.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/gdbusaddress.c b/gio/gdbusaddress.c
index 0044cd3..207838c 100644
--- a/gio/gdbusaddress.c
+++ b/gio/gdbusaddress.c
@@ -1331,7 +1331,7 @@ g_dbus_address_get_for_bus_sync (GBusType       bus_type,
 
       if (ret == NULL)
         {
-          ret = g_strdup ("unix:path=/var/run/dbus/system_bus_socket");
+          ret = g_strdup ("unix:path=/run/dbus/system_bus_socket");
         }
       break;
 
-- 
2.31.0

